packages/react-components/src/components/AppFrame/AppFrame.mdx

```diff
@@ -160,6 +160,74 @@ const SideNavigationSwitcher: FC = () => {
 };
 ```
 
+### Top bar <a id="TopBar" />
+
+The top bar is a component designed to display alerts and important messages to the user. 
+
+It has the following properties:
+
+- `children`, notably `NavigationTopBarAlert` (`NavigationTopBar.Alert`)
+- `title` - the title of the top bar. This is optional and will be displayed under the alerts.
+
+```jsx
+import { AppFrame, Navigation, NavigationTopBar, NavigationTopBarAlert } from '@livechat/design-system-react-components';
+
+<AppFrame
+  navigation={<Navigation>...</Navigation>}
+>
+  <NavigationTopBar>
+    <NavigationTopBarAlert kind="info">
+    Hello, this is an alert!
+    </NavigationTopBarAlert>
+  </NavigationTopBar>
+  App content
+</AppFrame>
+```
+
+
+### Top bar alert <a id="TopBarAlert" />
```

Comment:
We can include this section in "How to build the navigation" navigation links (line 20)